components/dialogs/AuthDialog.vue

```diff
@@ -92,7 +92,7 @@
               :is-sign-in="isSignIn"
               :language="getCurrentLocale"
               :redirect-url="getAuthCoreRedirectUrl"
-              :social-login-pane-style="socialLoginPanePosition"
+              :socialLoginPaneOption="socialLoginPaneOption"
```

Comment:
The props name is changed?

Reply:
yes, 2 diff prop, but probably should be named with `-` instead of camel